obs/preinstallimage-bios.sh.in

```diff
@@ -986,11 +986,13 @@ cat > /etc/default/ip6tables <<[eof]
 COMMIT
 [eof]
 
-# Accept tcp port 5005 for devel image (append files)
+# Accept tcp port 1099 and 5005 for devel image (append files)
```

Comment:
Perhaps say something about why to open these port => for java part (JMX + JVM debug).